### ISSUE TITLE
Staging Review finding: Combined statement doesn't indicate zero balance

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#107823]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Financial Management
**Product:** Debt Portal
**Feature:** One VA Debt Letter

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @allison0034 (Design), @briandeconinck (Accessibility)

### Description

On the combined statement web page, if there is $0.00 due for a copay or a debt, there's no affirmative indication that there is $0.00 due. The debt and bill overview and the combined statement PDF, however, both affirm that no balance is due and/or show a balance due of $0.00. For consistency and clarity if would be useful to indicate on the combined statement page whenever there is a zero balance (rather than saying nothing at all).

We should be internally consistent so that folks see the same information on the combined statement page and are able to make informed choices about what to do next. That consistency helps reduce cognitive load, minimizing the effort it takes for users to figure out their debt obligations.

<details><summary>Link, screenshot or steps to recreate</summary>

to be added

</details>

## Recommended action

Update combined statement page content for copay and/or debts so that when there is $0.00 due, the page says there is no money due.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by HeatherRienks** [comment]
Feedback has been addressed.
